.github/ISSUE_TEMPLATE/new_destination_plugin.yml

```diff
@@ -1,13 +1,13 @@
-name: 🔄 New Destination Plugin
```

Comment:
This file should also be renamed for consistency

Reply:
Done!